VideoLocker/AndroidManifest.xml

```diff
@@ -71,7 +71,7 @@
 
     <application
         android:allowBackup="true"
-        android:supportsRtl="false"
+        android:supportsRtl="@bool/supports_rtl"
```

Comment:
Of course it won't change dynamically, since the manifest is only read on application start-up. If we want to support run-time language changes that's a bigger task.